packages/@webex/contact-center/babel.config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "presets": [
```

Comment:
is there a way to avoid babel for this package ?

Reply:
Without Babel, the build gets impacted. Tried it when working on BYODS SDK, build started to fail

Reply:
i think there will be tooling changes needed to fix, just take a note and we can fix it later

Reply:
ok will have a story created to address this

Reply:
What does babel do and what happens when we remove it?